src/System.Reflection.TypeExtensions/tests/ConstructorInfo/ConstructorInfoInvokeArrayTests.cs

```diff
@@ -0,0 +1,269 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    public class ConstructorInfoInvokeArrayTests
+    {
+        [Fact]
+        public void Invoke_SZArrayConstructor()
+        {
+            Type type = Type.GetType("System.Object[]");
+            ConstructorInfo[] constructors = type.GetConstructors();
+            Assert.Equal(1, constructors.Length);
+
+            ConstructorInfo constructor = constructors[0];
+            int[] blength = new int[] { -100, -9, -1 };
```

Comment:
I know you just copied what was already there, but any idea what the 'b' in "blength" means?